src/Validation/Validator.php

```diff
@@ -556,9 +573,13 @@ protected function _processRules($field, ValidationSet $rules, $data, $newRecord
                 continue;
             }
 
-            $errors[$name] = $message;
             if (is_string($result)) {
                 $errors[$name] = $result;
+            } elseif ($this->_useI18n) {
+                $args = $rule->get('pass');
+                $errors[$name] = __d($this->_validationDomain, $name, $this->_translateArgs($args));
```

Comment:
Is that such a bad thing @ADMad? I thought the purpose of this PR was just to have the error messages indexed by the rule name. I don't think bringing `validationDomain` back is a good idea.